src/poetry/console/commands/command.py

```diff
@@ -17,6 +21,17 @@ class Command(BaseCommand):  # type: ignore[misc]
 
     _poetry: Poetry | None = None
 
+    def run(self, io: IO) -> int:
+        """
+        Temporarily fix for issue where io.input.stream is unset by cleo.
+        """
+        # FIXME: Remove method after upstream merge for cleo
+        # https://github.com/sdispater/cleo/pull/135
+        if io.input.stream is None:
+            io.input.set_stream(sys.stdin)
+        status_code: int = super().run(io)
```

Comment:
Is that extra variable needed? can't we just `return super().run(io)`?

Reply:
mypy complains about the type for `super.run()` being `Any` if I return it directly, so I wrapped it into an explicit cast here. I could swap to a mypy flag if that works better, though.

Reply:
No, lets leave like this.